### ISSUE TITLE
pip and conda seems not to work after a System Update

### ISSUE THREAD

**Issue by soma2000-lang** [pop-os/pop#2811]
![image](https://user-images.githubusercontent.com/56045049/221340140-72cbd8b2-06ee-424e-96b3-2446f6d43fff.png)
![image](https://user-images.githubusercontent.com/56045049/221340381-b67e1988-f802-4825-833a-0f4cae1652b0.png)

**Comment by soma2000-lang** [comment]
I have tried all the solution s mentioned in https://github.com/conda/conda/issues/11795 and https://stackoverflow.com/questions/25981703/pip-install-fails-with-connection-error-ssl-certificate-verify-failed-certi
but nothing seems to work.Any help regarding this matter would be really appreciated

**Comment by soma2000-lang** [comment]
cc @jackpot51

**Comment by DanielJoyce** [comment]
It says in your errors "OpenSSL is not installed on this machine", have you installed the libs, etc?

Openssl is not a conda package

Try 

```
$ apt-get install libssl-dev libffi-dev
```

**Comment by soma2000-lang** [comment]
@DanielJoyce Yes I have tried yet the same problem

**Comment by soma2000-lang** [comment]
I have almost tried all the solutions mentioned in https://stackoverflow.com/questions/3016956/how-do-i-install-the-openssl-libraries-on-ubuntu.But nothing seems to work.

**Comment by leviport** [comment]
> `cc @jackpot51 @burden @mmstick @Drakulix @crawfxrd @`

Hey, don't do that. We see all open issues. There's no need to annoy every System76 employee you can find on the internet.

**Comment by soma2000-lang** [comment]
@leviport I think I am facing some serious issue,and my important projects are being affected due to this issue.Its something urgent ,that-s why I just tagged.

**Comment by leviport** [comment]
@soma2000-lang I can tell that it's urgent for you, but I'm still telling you not to do that. You're tagging people that have nothing to do with this and cannot help you fix it. This isn't polite etiquette, and I cannot allow it to continue.

We do not typically work on the weekend. I understand waiting can be frustrating, but sometimes that's the only thing that can be done.

**Comment by soma2000-lang** [comment]
@isantop I executed the following commands
rm -rf venv
$ sudo apt-get update
$ sudo apt-get -y upgrade
 sudo apt-get install -y python3-pip
pip3 install package_name


Still I am getting the same error and the package is not being istalled

**Comment by isantop** [comment]
Does pip work if you aren't running in any venv? I.e. if you open a fresh terminal and attempt to install a package from pip?

**Comment by soma2000-lang** [comment]
@isantop Yes if I open a fresh terminal and ttry to install a package then also same problem

**Comment by DanielJoyce** [comment]
are the required libraries installed on your system? Pip can not install them,
